Use org auth token in JS and Sentry CLI docs (#7714)
getsentry/sentry-docs: src/platform-includes/sourcemaps/upload/webpack/javascript.mdx

> Learn more about configuring the plugin in our [Sentry webpack plugin documentation](https://www.npmjs.com/package/@sentry/webpack-plugin).

Reviewer: ```suggestion

```

> To upload source maps you have to configure an auth token.
> Auth tokens can be passed to the plugin explicitly with the `authToken` option, with a `SENTRY_AUTH_TOKEN` environment variable, or with a `.env.sentry-build-plugin` file in the working directory when building your project.
> You likely want to add the auth token as an environment variable to your CI/CD environment.

Reviewer: ```suggestion
To upload source maps you have to configure an auth token.
Auth tokens can be passed to the plugin explicitly with the `authToken` option, with a `SENTRY_AUTH_TOKEN` environment variable, or with an `.env.sentry-build-plugin` file in the working directory when building your project.
We recommend you add the auth token to your CI/CD environment as an environment variable.

Learn more about configuring the plugin in our [Sentry webpack plugin documentation](https://www.npmjs.com/package/@sentry/webpack-plugin).
```

<OrgAuthTokenNote />

> <SignInNote />
> 

Reviewer 2: ```suggestion
```
